### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by hainm**
copy from https://github.com/arose/nglview/releases/tag/v0.5

### CODE REVIEW

**Comment by hainm** [comment]
@arose I went over the commit log and added the LOG to https://github.com/arose/nglview/releases/tag/v0.5. Now I just copy them to `CHANGELOG.md`.
Please let me know if I miss anything.

**Comment by arose** [comment]
ok, will expand it a bit

**Comment by hainm** [comment]
I just updated few points from your email: https://github.com/arose/nglview/pull/272/commits/25c1fe8e69f67b641788349e8ef7aa2bd0a698ff

You can update the `README.md` directly on github in `hainm-patch-1` branch.

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,38 +3,38 @@ Changelog
...
-* ADD: shortcut for add_representation (add_cartoon, add_rope, ...)
-* ADD: `ParmEdTrajectory` adaptor and `show_parmed`
-* ENH: `add_trajectory` can accept `pytraj.Trajectory`, `mdtraj.Trajectory`, `MDAnalysis.Universe`, `parmed.Structure`
-* ENH: add_representations can accept both snake and CamelKeyword
+
+* ADD: `view.delay` property to adjust trajectory playback
+* ADD: `show_url` function for loading & viewing from an url
+* ADD: `view.hide()` and `view.show_only()` convenience methods
+* ADD: `show_rdkit` function to view rdkit Mol objects
+* ADD: `view.clear()` for clearing all representations
```
Comment (hainm) [inline CHANGELOG.md]: Actually `view.clear(component=index)`. :D (default index=0)

which is same as `view.add_representations(component=0)`

Comment (arose) [inline CHANGELOG.md]: yeah, right, just trying to be succinct here

**Pull request merged**